packages/sdk-core/src/modules/metrics/BacktraceMetrics.ts

```diff
@@ -38,6 +38,9 @@ export class BacktraceMetrics implements BacktraceModule {
         private readonly _summedEventsSubmissionQueue: MetricsQueue<SummedEvent>,
         private readonly _uniqueEventsSubmissionQueue: MetricsQueue<UniqueEvent>,
     ) {
+        this.metricsHost = this._options.metricsSubmissionUrl ?? this.DEFAULT_SERVER_URL;
```

Comment:
do we need this change?

Reply:
yes, it errors out, presumably because we use `this._options` here